fab/fabfile.py

```diff
@@ -483,6 +486,21 @@ def record_successful_deploy(url):
         })
 
 
+@task
+@roles(ROLES_DB_ONLY)
+def set_in_progress_flag():
+    sudo('redis-cli SET {} true'.format(IN_PROGRESS_FLAG))
+
+    # Set deploy flag to expire in 30 minutes
+    sudo('redis-cli EXPIRE {} {}'.format(IN_PROGRESS_FLAG, 60 * 30))
```

Comment:
I think 30 minutes is too much (having celery be down for 30 minutes without knowing would be bad).  Can we set the flag right before we stop all celery services and set the timeout to 5-10 minutes?